optimum/habana/accelerate/accelerator.py

```diff
@@ -458,7 +467,11 @@ def _prepare_deepspeed(self, *args):
 
         is_dataloader_present = any(isinstance(obj, torch.utils.data.DataLoader) for obj in args)
         result = [
```

Comment:
The changes are specific to deepspeed. If you really don't need to use deepspeed, you could avoid making these changes for now.

Reply:
We also tried with mixral8-7b, and it uses deepspeed